src/ContinuosBondingERC20Token.sol

```diff
@@ -231,15 +230,4 @@ contract ContinuosBondingERC20Token is ERC20, ReentrancyGuard {
 
         emit PairCreated(currentEth, currentTokenBalance, liquidity, address(lpToken));
     }
-
-    function _update(address from, address to, uint256 value) internal virtual override {
```

Comment:
The change looks good. As discussed in teams, we need to take care of following edge case. 

> if someone already created the pool and added liquidity and got some shares with bad price ratio, the contract once it reaches liquditiy goal will also be adding liquidity at imbalanced price. So, the contract won't be able to add liquidity in the amount we desire in such cases.